tests/hash-graph-test-data/rust/src/data_type/object_v2.json

```diff
@@ -4,7 +4,7 @@
   "$id": "https://blockprotocol.org/@blockprotocol/types/data-type/object/v/2",
   "allOf": [
     {
-      "$ref": "https://blockprotocol.org/@blockprotocol/types/data-type/value/v/1"
+      "$ref": "https://blockprotocol.org/@blockprotocol/types/data-type/object/v/1"
```

Comment:
I don't know why this did not error before, but `object/v/2` is not on the whitelist of allowed primitives. To circumvent this, we just inherit from `object/v/1`.

Reply:
Fixed in #5308